package.json

```diff
@@ -21,8 +21,8 @@
 		"coverage-unit": "jest --config test/.jest.json",
 		"doc": "npm run doc-generate && npm run doc-sanitize-avsc && npm run doc-sanitize-node",
 		"doc-generate": "typedoc --gitRevision master --ignoreCompilerErrors --out doc src/**/* --readme readme.md",
-		"doc-sanitize-avsc": "sed -E -i '' 's|'$PWD'/node_modules/avsc/types/index.d.ts:([0-9]*)|<a href=\"https://github.com/mtth/avsc/blob/master/types/index.d.ts#L\\1\">avsc/types/index.d.ts:\\1</a>|g' $(find doc -type f -regex '^doc/[a-zA-Z/._]*.html$')",
-		"doc-sanitize-node": "sed -E -i '' 's|'$PWD'/node_modules/@types/node/events.d.ts:([0-9]*)|<a href=\"https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/events.d.ts#L\\1\">node/events.d.ts:\\1</a>|g' $(find doc -type f -regex '^doc/[a-zA-Z/._]*.html$')",
+		"doc-sanitize-avsc": "sed -E -i '' 's|'$PWD'/node_modules/avsc/types/index.d.ts:([0-9]*)|<a href=\"https://github.com/mtth/avsc/blob/master/types/index.d.ts#L\\1\">avsc/types/index.d.ts:\\1</a>|g' $(find doc -type f -regex '^doc/[a-zA-Z1-9/\\._-]*.html$')",
```

Comment:
Should the regex be 0-9? Seems odd that we'd let any number of zeroes through, when all the other digits are so unwelcome.

Reply:
Yes, this should be `0-9` 👍 We only had file names ending `-1` so I hadn't included 0.